sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/HttpPipelineAdapter.java

```diff
@@ -57,4 +84,78 @@ public IHttpResponse send(HttpRequest httpRequest) throws Exception {
                 })))
             .block();
     }
+
+    private void logAccounIdentifiersIfConfigured(String body) {
+        if (identityClientOptions != null
```

Comment:
Should this be `identityClientOptions == null || !identityClientOptions.getI...`?

Reply:
Good catch.

Reply:
Updated it, although the input as per current design won't be null, but as an isolated class, its safe to add this check.